source/collection-management/tests/timeseries-collection.yml

```diff
@@ -109,7 +114,6 @@ tests:
               command:
                 insert: *collection0Name
                 documents: *docs
-    outcome:
-    - collectionName: *collection0Name
-      databaseName: *database0Name
-      documents: *docs
+
+   # Expect no particular outcome from the collection since ordering of
+   # documents may vary given duplicate _ids.
```

Comment:
I realize you can't use `outcome` here, but you can still do an assertion with a sorted `find` operation and `expectResult`. I've done this elsewhere in other tests where insert order might be ambiguous.

Reply:
Gotcha, removed this comment in favor of a sorted `find` operation with `expectResult: *docs`.